manifest.yaml

```diff
@@ -69,33 +69,33 @@ interfaces:
     protocols:
       - tcp
       - monero
-  rpc:
+  rpc-restricted:
     name: RPC Interface
     #description: Listens for JSON-RPC commands
     description: Do not use the URL below.  Instead, look in Monero's Properties section for your interface addresses.
     tor-config:
       port-mapping:
         18081: "18081"
```

Comment:
should this be 18089

Reply:
Very right, thanks for catching that one.  Fixed by [ced8135](https://github.com/kn0wmad/monerod-startos/pull/32/commits/ced81350fc70b1c3249a291b277cf1a359fce866)